imessage/bluebubbles/interface.go

```diff
@@ -35,11 +35,23 @@ type ChatQueryResponse struct {
 	Metadata PageMetadata `json:"metadata"`
 }
 
+type ChatResponse struct {
+	Status  int64  `json:"status"`
+	Message string `json:"message"`
+	Data    *Chat  `json:"data,omitempty"`
+}
+
 type Chat struct {
 	// TODO How to get timestamp
-	GUID           string `json:"guid"`
-	ChatIdentifier string `json:"chatIdentifier"`
-	GroupId        string `json:"groupId"`
+	GUID           string        `json:"guid"`
+	ChatIdentifier string        `json:"chatIdentifier"`
+	GroupId        string        `json:"groupId,omitempty"`
```

Comment:
This field was already there, but as a side note, `ID` should be uppercase https://go.dev/wiki/CodeReviewComments#initialisms